package.json

```diff
@@ -15,7 +15,7 @@
   "license": "ISC",
   "homepage": "https://bitbucket.org/edunext/eox-nelp#readme",
   "dependencies": {
-    "@edunext/frontend-essentials": "^1.6.1",
+    "@edunext/frontend-essentials": "^1.7.0-rc",
```

Comment:
I think this will be updated after all process is completed. I mean essentials bump?

Reply:
done

Reply:
done